### PR TITLE
Remove override for narrow.

### PR DESCRIPTION
**Pull request by ailzhang**
`XLATensor::narrow` is still used by `Select`, so we're not removing it at tensor level.

### CODE REVIEW

File: torch_xla/csrc/aten_xla_type.cpp [hunk 1]
```diff
@@ -1999,13 +1999,6 @@ at::Tensor& AtenXlaType::mv_out(at::Tensor& out, const at::Tensor& self,
   return out;
 }
 
-at::Tensor AtenXlaType::narrow(const at::Tensor& self, int64_t dim,
-                               int64_t start, int64_t length) {
-  XLA_FN_COUNTER("xla::");
-  return bridge::AtenFromXlaTensor(
-      XLATensor::narrow(bridge::GetXlaTensor(self), dim, start, length));
```
Comment (dlibenzi) [inline torch_xla/csrc/aten_xla_type.cpp]: The `XLATensor::narrow()` still have uses?

Comment (ailzhang) [reply torch_xla/csrc/aten_xla_type.cpp]: Yup it's used by Select. :D

Comment (ailzhang) [reply torch_xla/csrc/aten_xla_type.cpp]: https://github.com/pytorch/xla/blob/de07b24d10b6c53de8e603c5756e4dda6d42cb35/torch_xla/csrc/tensor_ops.cpp#L176

Comment (dlibenzi) [reply torch_xla/csrc/aten_xla_type.cpp]: 👻

**Pull request merged**